OpenOversight/app/templates/partials/officer_general_information.html

```diff
@@ -0,0 +1,48 @@
+<div class="thumbnail">
+    <div class="caption">
+        <h3>
+            General Information
+            {% if is_admin_or_coordinator %}
+            <a href="{{ url_for('main.edit_officer', officer_id=officer.id) }}">
+                <i class="fa fa-pencil-square-o" aria-hidden="true"></i>
+            </a>
+            {% endif %}
+        </h3>
+        <table class="table table-hover">
+            <tbody>
+                <tr>
+                    <td><b>Name</b></td>
+                    <td>{{ officer.full_name() }}</td>
+                </tr>
+                <tr>
+                    <td><b>OpenOversight ID</b></td>
+                    <td>{{ officer.id }}</td>
+                </tr>
+                <tr>
+                    <td><b>Unique Internal Identifier</b></td>
+                    <td>{{ officer.unique_internal_identifier }}</td>
+                </tr>
+                <tr>
+                    <td><b>Department</b></td>
+                    <td>{{ officer.department.name }}</td>
+                </tr>
+                <tr>
+                    <td><b>Race</b></td>
+                    <td>{{ officer.race_label() }}</td>
+                </tr>
+                <tr>
+                    <td><b>Gender</b></td>
+                    <td>{{ officer.gender_label() }}</td>
+                </tr>
+                <tr>
+                    <td><b>Birth Year (Age)</b></td>
```

Comment:
Not a note for this PR but...I feel like these function calls should probably be properties rather than functions :thinking: Things like `race_label`, `gender_label`, `full_name`, etc.